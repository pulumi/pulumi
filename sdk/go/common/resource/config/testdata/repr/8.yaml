my:arrayKey:
- a
- b
- c
