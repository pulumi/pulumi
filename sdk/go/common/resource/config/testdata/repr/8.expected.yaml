decrypt:
  my:arrayKey: '["a","b","c"]'
paths:
  my:arrayKey:
    value:
    - a
    - b
    - c
    string: '["a","b","c"]'
    redacted: '["a","b","c"]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAAKf+EJgADBnN0cmluZwwDAAFhBnN0cmluZwwDAAFiBnN0cmluZwwDAAFj
    secure: false
    isObject: true
  my:arrayKey[0]:
    value: a
    string: a
    redacted: a
    object: DhAABnN0cmluZwwDAAFh
    secure: false
    isObject: false
  my:arrayKey[1]:
    value: b
    string: b
    redacted: b
    object: DhAABnN0cmluZwwDAAFi
    secure: false
    isObject: false
  my:arrayKey[2]:
    value: c
    string: c
    redacted: c
    object: DhAABnN0cmluZwwDAAFj
    secure: false
    isObject: false
